app/helpers/forms_helper.rb

```diff
@@ -1,5 +1,6 @@
 module FormsHelper
-  def form_field_tag(&)
-    tag.div class: "form-field", &
+  def form_field_tag(&block)
```

Comment:
@JoshAntBrown  Might have made things more complicated again as I couldn't use the form_field_tag without added the &block unless I've missed something. Happy to change this in favour of directly using something like: 

`<div class="form-fields">content</div>`

Reply:
I was going to ask about this, what was the issue you had?

I've just pulled your branch down to my machine and reverted this back to the original implementation and it looks like it works for me.

Reply:
Ahh yes you're right it works for me reverting now, I was fixing merge conflicts whilst looking at this and couldn't get it to work without explicitly setting the context. Reverting now